mockOSApi.v2/Controllers/ProcessController.cs

```diff
@@ -12,20 +18,41 @@ public class ProcessController : Controller
     private readonly ILogger<ProcessController> _logger;
     private readonly IProcessService _processService;
 
-    public ProcessController(ILogger<ProcessController> logger, IProcessService processHandler)
+    private readonly IAuthentication _authorizationService;
+
+    public ProcessController(ILogger<ProcessController> logger, IProcessService processHandler, IAuthentication authorizationService)
     {
         _logger = logger;
         _processService = processHandler;
+        _authorizationService = authorizationService;
+    }
 
+    private string? FetchTokenFromHeaders()
+    {
+        string token;
+        try
+        {
+            token = HttpContext.Request.Headers["Authorization"].ToString().Split(" ")[1];
```

Comment:
So you are assuming it's in `Bearer <token>` format here?

Reply:
yes, added check to see if the input uses the Bearer <token> schema